node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -667,6 +772,22 @@ var _isEmailAddressUpdate = function(principalId, profileFields, callback) {
  * @param  {Group|User}     callback.principal  The requested principal
  * @api private
  */
+var getPrincipalFromCassandra = module.exports.getPrincipalFromCassandra = function(principalId, callback) {
```

Comment:
Why is this needed? Apart from anything else, the name is deceptive; it only returns the principal if it's a user. What's the added benefit to using this over the existing methods?